src/core/Private/documentrw.cpp

```diff
@@ -125,6 +253,25 @@ void Document::writeDoxumentXEV(QDomDocument& doc) const {
     }
     document.appendChild(bBrushes);
 
+    auto gridSettings = doc.createElement("GridSettings");
+    const auto& grid = mGridController.settings;
+    gridSettings.setAttribute("sizeX", QString::number(grid.sizeX));
+    gridSettings.setAttribute("sizeY", QString::number(grid.sizeY));
+    gridSettings.setAttribute("originX", QString::number(grid.originX));
+    gridSettings.setAttribute("originY", QString::number(grid.originY));
+    gridSettings.setAttribute("snapThresholdPx", QString::number(grid.snapThresholdPx));
+    gridSettings.setAttribute("enabled", grid.enabled ? "true" : "false");
+    gridSettings.setAttribute("show", grid.show ? "true" : "false");
+    gridSettings.setAttribute("majorEveryX", QString::number(grid.majorEveryX));
+    gridSettings.setAttribute("majorEveryY", QString::number(grid.majorEveryY));
+    // Legacy attribute for older consumers expecting a unified value.
```

Comment:
?

Reply:
Old code,  it wanted to keep compatibility with "majorEvery" as in my code it existed:
```
// Legacy attribute for older consumers expecting a unified value.
    gridSettings.setAttribute("majorEvery", QString::number(grid.majorEveryX));
    ```
removing it, we don't need it.